docs/guides/ecosystem/docker.md

```diff
@@ -0,0 +1,109 @@
+---
+name: Containerize a bun application using Docker
+---
+
+[Docker](https://www.docker.com) is a set of platform as a service products that use OS-level virtualization to deliver software in packages called containers. The service has both free and premium tiers.
+
+---
+
+For this first you need to install Docker Desktop to can build docker images, this can be downloaded from their [official web application](https://www.docker.com/products/docker-desktop/)
+
+---
+
+Now that you have docker, we can start writing the `Dockerfile`, this file should be in the root of your application, a basic `Dockerfile` that uses a build stage can be this, if you don't need a build stage you can remove it from the file.
+
+```dockerfile#Dockerfile
+# * Here we use the official image of bun which already has bun installed, you can specify the exact version
+# * Like 1.0.6-alpine or always get the latest version with alpine e.g. FROM oven/bun:alpine as base
+# * Getting always the latest is not such a great idea as it might introduce breaking changes
+# * You have 2 more options which are 1-alpine and 1.0-aline, which will use the version 1.x.x (Latest minor version) or 1.0.x (Latest revision version)
+FROM oven/bun:1.0.6-alpine as base
```

Comment:
I don't think alpine should be the default in the guide. Alpine is technically hacky, since we don't support MUSL.

Reply:
Would debian be better? I usually choose alpine cause it has a smaller size

Reply:
I added debian @Electroid